peril-settings.json

```
@@ -8,6 +8,9 @@
     "rules": {},
     "repos": {
         "wordpress-mobile/WordPress-iOS": {
+            "merge_group": [
+                "Automattic/peril-settings@org/pr/ios-macos.ts"
+            ],
             "pull_request": [
                 "Automattic/peril-settings@org/pr/ios-macos.ts",
                 "Automattic/peril-settings@org/pr/milestone.ts"
```

I was tempted to do the following, but wasn't game enough to try it while "in a rush" to fix the merge queue

```suggestion
            "merge_group, pull_request": [
                "Automattic/peril-settings@org/pr/ios-macos.ts"
            ],
            "pull_request": [
                "Automattic/peril-settings@org/pr/milestone.ts"
```

It seems safe enough to me, but maybe I'll wait till after knowing the current setup works for merge queue

---

Yeah those should be equivalent, but I like that you kept them separate for the time being, makes it easier to revert, or to replicate for other repos, depending on how the experiment goes.